model/database/cosmosdb-metrics.yaml

```diff
@@ -26,3 +26,68 @@ groups:
       - ref: server.port
         requirement_level:
           conditionally_required: If using a port other than the default port for this DBMS and if `server.address` is set.
+
+  - id: metric.db.client.cosmosdb.request.duration
+    type: metric
+    metric_name: db.client.cosmosdb.request.duration
+    brief: "End to end Latency of Azure Cosmos DB Backend Service network call"
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    extends: attributes.db.cosmosdb.network.minimal
+
+  - id: metric.db.client.cosmosdb.request.body.size
+    type: metric
+    metric_name: db.client.cosmosdb.request.body.size
+    brief: "Size of request object sent to Azure Cosmos DB backend"
+    instrument: histogram
+    unit: "{bytes}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.network.minimal
+
+  - id: metric.db.client.cosmosdb.response.body.size
+    type: metric
+    metric_name: db.client.cosmosdb.response.body.size
+    brief: "Size of response received from Azure Cosmos DB backend"
+    instrument: histogram
+    unit: "{bytes}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.network.minimal
+
+  - id: metric.db.client.cosmosdb.request.channel_aquisition.duration
+    type: metric
+    metric_name: db.client.cosmosdb.request.channel_aquisition.duration
+    brief: "Time taken in creating connection, It is only applicable in Direct Mode."
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    extends: attributes.db.cosmosdb.network.minimal
+
+  - id: metric.db.server.cosmosdb.request.duration
+    type: metric
+    metric_name: db.server.cosmosdb.request.duration
```

Comment:
Are we going to report it on the client side?

Reply:
yes, we are going to report, backend latency to customer.

Reply:
Let's document it then - that this is the latency returned in response and reported from the client. It will not be reported if response was not received